src/fstext/determinize-star-inl.h

```diff
@@ -468,99 +469,221 @@ template<class Arc> class DeterminizerStar {
   // Called by ProcessSubset.
   // Has no side effects except on the repository.
 
+  struct EpsilonClosureInfo {
+    EpsilonClosureInfo() {}
+
+    EpsilonClosureInfo(Element e, Weight w, bool i, int d = 0) :
```

Comment:
Better to make this constructor take const Element &e, const Weight &w: avoid constructor calls.